### PR TITLE
Updated fields.py to satisfy Django 1.11 changes

### PR DESCRIPTION
**Pull request by beardo01**
Note that I haven't done significant testing. It appears to be working.

### CODE REVIEW

File: django_encrypted_filefield/fields.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def save(self, name, content, save=True):
 
     def _get_url(self):
         return reverse(FETCH_URL_NAME, kwargs={
-            "path": FieldFile._get_url(self)
+            "path": super(EncrpytionMixin, self).url
```
Comment (danielsamuels) [inline django_encrypted_filefield/fields.py]: Typo, should be `EncryptionMixin`

File: django_encrypted_filefield/fields.py~ [hunk 2]
```diff
@@ -0,0 +1,70 @@
+from io import BytesIO
```
Comment (danielsamuels) [inline django_encrypted_filefield/fields.py~]: I'm not sure you intended to include this temp file.

Comment (beardo01) [reply django_encrypted_filefield/fields.py~]: Correct! My bad.

**Comment by beardo01** [comment]
There we go, that should fix those errors. Proves that you shouldn't make PR at stupid times of the night!

**Comment by solvire** [comment]
Thanks for finding the fix @beardo01 - any idea what's holding up the pull request? Testing?

**Comment by beardo01** [comment]
Unsure at this stage man.  I was hoping that Daniel would add it in.

**Comment by ashwch** [comment]
Hi @danielquinn, any chances of this PR getting merged anytime soon?

**Comment by danielquinn** [comment]
Wow.  I'm so sorry guys.  I hadn't realised that I'd left this open.  Merging now and I'll update PyPi shortly.

**Comment by beardo01** [comment]
@danielquinn Cheers! Thanks for the update.

**Pull request merged**